### ISSUE TITLE
Problem loading the geom-edges.R script

### ISSUE THREAD

**Issue by evinhas** [briatte/ggnetwork#2]
Hi there,

Please, could you review the error I am getting when I am trying to load the *.R files?

> geom-edges.R :Error in ggplot2::ggproto("StatEdges", Stat, compute_layer = function(data,  : 
>  objet 'Stat' is not found

I was checking on geom-edges.R, and there is neiter declaration or assignment for object Stat.

Note: I was trying to run the following "for" loop for getting load all the *.R files are in your package:

> #path to the R files inside the package I've just download from GitHub
> path="C:\\DATOS\\GNU-R\\briatte-ggnetwork-f58c4d9\\briatte-ggnetwork-f58c4d9\\R"

> #A for loop to get all files loaded
> for (nm in list.files(path, pattern = "[.][RrSsQq]$")) {
>       if(trace) cat(nm,":")
>      source(file.path(path, nm))
>     if(trace) cat("\n")
>  }

**Comment by briatte** [comment]
Could you please also post your session information?

```R
?sessionInfo
```

**Comment by evinhas** [comment]
Hi briatte,

I found a workaround: instead of use a for loop for dynamically sourcing the files, I opted to hardcoded the files and paths, without any loop, and it worked.

So we can close the issue, as it seems something related to the loop. The loop was copy&pasted from internet, to avoid use administrator privilegies (as it is a laptop using windows from a company) for installing the package from github, as I am behind a proxy using my client network, and I couldn't reach github.

Anyway, thanks in advance.

Apologize for the inconveniences.
